index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" type="text/css" href="main.css">
+    <script src="snake.js"></script>
+
+    <title>Document</title>
+</head>
+<body>
+    <canvas id="container" width="500" height="400"></canvas>    
```

Comment:
Id `container` doesn't have any meaning in out context, please use something related to project (hint: maybe game name)